tests/utils/wdio-utils.js

```diff
@@ -0,0 +1,38 @@
+const SUITES = {
```

Comment:
I'm not so sure about the location of this file, why is it in /utils when it's only useful for `default`. I think it would make more sense to rename this to `suites.js` or something and save it in the `/default` folder.